package.json

```diff
@@ -24,12 +24,12 @@
   "dependencies": {},
   "devDependencies": {
     "chai": "^3.4.1",
-    "chai-subset": "^1.1.0",
+    "chai-subset": "^1.2.2",
     "coveralls": "^2.11.4",
-    "esprima": "^2.7.0",
+    "cst": "^0.3.0",
```

Comment:
Some rules in `lib` explicitly require the `cst` module so it needs to be a dependency, not dev dependency.